index.html

```diff
@@ -735,93 +920,138 @@ <h2 class="subscribe-block__title title">
             alt=""
           />
 
-          <div class="footer-section__our-contacts">
-            <a class="footer-section__logo logo link-black" href="#">
-              J<span class="decor-orange">o</span>bply
+          <div class="footer__our-contacts">
+            <a class="logo link-black" href="#">
+              J<span class="decor_orange">O</span>BPLY
             </a>
-            <p class="footer-section__address">
-              152/3 , Western Road, New York
-            </p>
-            <p class="footer-section__call-us hover-orange">
+            <p class="footer__address">152/3 , Western Road, New York</p>
+            <a href="tel:+99-8569-5234" class="footer__call-us hover_orange">
```

Comment:
```suggestion
            <a href="tel:+99-8569-5234" class="link_smth text_smth">
```
Inspect the design, find all the text and links variants, and create universal classes for links and texts and add modifiers for them if needed. Use these classes for texts and links instead of giving personal class names for every tag with such content.

Class names above are just a suggestion, use suitable class names.